.github/workflows/deploy-Frontend.yml

```diff
@@ -18,6 +18,16 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+      
+      - name: Install dependencies
+        run: |
+          cd frontend
+          npm install --legacy-peer-deps
```

Comment:
I've tried using `npm ci` for `activity-tracking` like [here](https://github.com/VathsalaAchar/MLA-pilot-group4/blob/375f33a8cead71420159d02592985705cb0d4a6c/.github/workflows/deploy-ActivityTracking.yml#L36)
This could be useful here as well.